Uppgift12-Garage20/Views/ParkedVehicles/Create.cshtml

```diff
@@ -12,11 +12,13 @@
     <div class="col-md-4">
         <form asp-action="Create">
             <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <!--  include a dropdown list for selecting the vehicle type -->
             <div class="form-group">
                 <label asp-for="VehicleType" class="control-label"></label>
-                <input asp-for="VehicleType" class="form-control" />
+                <select asp-for="VehicleType" asp-items="@(new SelectList(Enum.GetValues(typeof(Uppgift12_Garage20.Models.VehicleType))))" class="form-control"></select>
```

Comment:
Just a suggestion, you could also write `asp-items="@Html.GetEnumSelectList<VehicleType>()"`
to make it shorter